doc/run/docker.md

```diff
@@ -20,45 +20,8 @@ You can use this Docker image for either local trying out or production deployme
 1. Type the below command to start the container:
 
    ```
-   docker run -it -p 8888:8888 sqlflow/sqlflow
+   docker run --rm -it -p 8888:8888 sqlflow/sqlflow
    ```
 
-1. Open a web browser, go to `localhost:8888`, open `tutorial_dnn_iris.ipynb` file, then you can
+1. Open a web browser, go to `localhost:8888`, open `iris-dnn.ipynb` file, then you can
    follow the tutorial and run the SQL statements to run the training and prediction.
-
-## Connect to Your Own Database
```

Comment:
I believe connect to your own database deserves a separate document like https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/run_with_hive.md. So I deleted this section.

Reply:
There seems have no documents to describe how to use a stand alone MySQL instance with SQLFlow, the document https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/mysql-setup.md seems outdated, we may still need a section to describe how to connect SQLFlow with already existed DB instances.

Maybe we can keep connect to mysql section and remove contents of Hive section and refer to the https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/run_with_hive.md?

Reply:
Sounds good.